include/mantella_bits/numericalAnalysis.hpp

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+// C++ standard library
+#include <functional>
+
+namespace mant {
+
```

Comment:
The above newline is superfluous and should be removed. Sadly, this cannot be enforced by clang-format yet.